gulpfile.js/build.js

```diff
@@ -355,24 +387,63 @@ function buildPages(done) {
         .pipe(gulp.dest(`${project.paths.PAGES_DEST}/shared`));
     },
     // eslint-disable-next-line prefer-arrow-callback
-    async function publishPages() {
-      if (!config.options.locales.includes(config.getDefaultLocale())) {
+    async function copyBuildFiles(done) {
+      if (!config.options?.locales?.includes(config.getDefaultLocale())) {
         console.log(
           'Skipping page publishing. Default language is not build, only:',
           config.options.locales
         );
-        return;
+        return done();
```

Comment:
curiosity: do you still need to call `done` when using returning promises?

Reply:
I _think_) it was broken without it. If memory serves, `done` needed to be within the function's arity, and thereby also needs to be called once added. `done()` returns a promise as well that gulp is expecting. But I could be misremembering it - gulp kicked my butt on this for a bit

Reply:
I've definitely been there as well....